docker/utils/v4r_and_openni_kinetic.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+echo "Installing ceres with dependencies" && \
```

Comment:
there is no need to have `&& \` after each statement. It is only needed for Dockefile.